app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -41,26 +55,57 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
+            <UserSummary user={user}>
+              <Container
+                disableGutters={true}
+                className={classes.statusLabelWrapper}
+              >
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="subtitle1"
+                  color="primary"
+                >
                   {hostingStatusLabels[user.hostingStatus]}
                 </Typography>
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="subtitle1"
+                  color="secondary"
+                >
+                  {meetupStatusLabels[user.meetupStatus]}
+                </Typography>
               </Container>
-            </div>
-
-            <ScoreBar value={user.communityStanding * 100}>
-              {COMMUNITY_STANDING}
-            </ScoreBar>
-            <ScoreBar value={user.verification * 100}>
-              {VERIFICATION_SCORE}
-            </ScoreBar>
-
-            {user.aboutMe.length < 300
-              ? user.aboutMe
-              : user.aboutMe.substring(0, 300) + "..."}
+              <Typography variant="h6" className={classes.about}>
```

Comment:
```suggestion
              <Typography component="h3" variant="h6" className={classes.about}>
```
This shouldn't be a `h6` since it skips a couple of heading levels which is bad practice for accessibility. Better to use a `h3` but style it like a `h6` for visual users.

Reply:
Good point, I didn't realise I could just specify component level while keeping variant styling 👍